### ISSUE TITLE
Redundand whitespace left by the rewriter

### ISSUE THREAD

**Issue by rkapka** [rubberduck-vba/Rubberduck#3850]
As mentioned in #3834, the rewriter leaves some redundand whitespace after removing pieces of code. The quick fix tests for the linked PR expect that whitespace, but it just doesn't feel right.

**Comment by retailcoder** [comment]
Would it be acceptable to run *indent procedure* to fix that?

**Comment by Vogel612** [comment]
Honestly: IMO it wouldn't. We don't want to force SmartIndenter use. The solution should be to update the grammar to parse whitespace in a way that makes the rewriter behaviour correct.

**Comment by retailcoder** [comment]
@Vogel612 agreed. However updating the grammar to deal with WS would be excruciatingly hard, and would break many places that deal with this parse tree quirk.

IMO the rewriter itself could smarten up, and verify whether there are extraneous WS tokens on the same line, that should be removed too.
